src/js/filter-form.js

```diff
@@ -69,17 +69,16 @@ class FilterForm {
 	 */
 	handleFormChangeEventImmediate(event) {
```

Comment:
should we preventDefault() on this, so it doesn't do a full page reload?

Reply:
I'll need to check this out locally and give it a proper test, might be able to simplify these different handler functions.